### PR TITLE
docs: change README

### PR DESCRIPTION
**Pull request by xyuuyaa**
## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

Changed the README. Add short description of program, Add features of program

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,23 @@
-# SoccerFields
+Here’s a more neutral version of the README text:
```
Comment (danieljancar) [inline README.md:1]: remove

File: README.md [hunk 2]
```diff
@@ -1,27 +1,23 @@
...
 
-## Code scaffolding
+-   **League Selection**: Choose a league and the time span of games.
+-   **Detailed Tables**: Displays data such as team name, rank, points, victories, and more.
+-   **Sorting Options**: Sort the table by points, goal difference, victories, and team names.
+-   **All-Leagues View**: View data for all leagues if no specific league is selected.
+-   **Data Download**: Download tables for later use.
 
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+This program is based on the M426 module "Git and Scrum" at BBZW Sursee.
```
Comment (danieljancar) [inline README.md:15]: This webapp is ba....

File: README.md [hunk 3]
```diff
@@ -1,27 +1,23 @@
-# SoccerFields
+Here’s a more neutral version of the README text:
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 18.0.1.
+# Soccer Fields
 
-## Development server
+This program contains fictional match results of European leagues for the 2023/2024 season. It calculates and displays results in a tabular format for each league.
```
Comment (danieljancar) [inline README.md:5]: This webapp contains...

**Review by danieljancar — CHANGES_REQUESTED** [review]
Looks alright as a starting point to work on the weekend. Some small changes needed tho 🙈

File: README.md (hunk 1 above)
Comment (KLGamet) [reply README.md:1]: Contents of the readme.md have already been updated in the branch "testing"

**Pull request closed without merging**